frame/multisig/src/lib.rs

```diff
@@ -524,6 +545,26 @@ impl<T: Trait> Module<T> {
 		T::AccountId::decode(&mut &entropy[..]).unwrap_or_default()
 	}
 
+	/// Place a call's encoded data in storage, reserving funds as appropriate.
+	fn store_call(who: T::AccountId, hash: &[u8; 32], data: Vec<u8>) -> DispatchResult {
+		if !Calls::<T>::contains_key(hash) {
+			let deposit = T::DepositBase::get()
+				+ T::DepositFactor::get() * BalanceOf::<T>::from(((data.len() + 31) / 32) as u32);
```

Comment:
Can you document how this formula came to be? Where do the magic numbers come from?

Reply:
The docs of `DepositFactor` read:
> The amount of currency needed per unit threshold when creating a multisig execution.
> 
> This is held for adding 32 bytes more into a pre-existing storage value.

The `DepositFactor` part of the formula just rounds up to the nearest 32-byte step before multiplying it to get the marginal term of the deposit.

The docs of `DepositBase` has been slightly updated to read:
> The base amount of currency needed to reserve for creating a multisig execution or to store a dispatch call for later.
> 
> This is held for an additional storage item whose value size is
> `4 + sizeof((BlockNumber, Balance, AccountId))` bytes and whose key size is
> `32 + sizeof(AccountId)` bytes.


```

Reply:
ok, so the 32 is just documented there, sorry for not checking beforehand.
:+1: